yivi-docs/blog/2025-7-03-yivi-portal-announcement/index.md

```diff
@@ -0,0 +1,49 @@
+---
+slug: 2025-july-portal-announcement
```

Comment:
Also missing the part that we want to cover the downstream costs of issuers with this.